packages/astro/src/core/ssr/index.ts

```diff
@@ -269,9 +296,9 @@ async function getHmrScript() {
 export async function ssr(ssrOpts: SSROptions): Promise<string> {
   try {
     const [renderers, mod] = await preload(ssrOpts);
-    return render(renderers, mod, ssrOpts);
+    return await render(renderers, mod, ssrOpts); // note(drew): without "await", errors won’t get caught by errorHandler()
```

Comment:
💣 this blew everything up (not `await`-ing). Find by @FredKSchott!

Reply:
oof, good catch